### ISSUE TITLE
More removable capabilities

### ISSUE THREAD

**Issue by bell07** [Gerold55/minetest-laptop#84]
We need better support for media in the Media Pack  https://forum.minetest.net/viewtopic.php?f=9&t=19580&p=310974#p310974
At the time all media are assigned to the "floppy" capability, that results in the punch card can be used to boot a system. We need the next capabilities
 - [x] usb - Used primary for storage, but can be used to store full system backup
-  [x] floppy - Used as storage, can be booted in live mode, in this case the storage is used as system storage
- [ ] tape - Should be usable for full system backups only, for old computers without usb
- [ ] cd - system backup or live boot. In case of live boot the system storage should be in "RAM-Disk"
- [ ] punched - ???

The assignment to the hardware will be:  tape and punched are assigned to the old "CUBE" and "Pentium 3", the cd to all other computers.

**Comment by Gerold55** [comment]
Computer punched card reader — a computer input device used to read executable computer programs and data from punched cards under computer control. Computer card punch — a computer output device that punches holes in cards under computer control.

**Comment by Grizzly-Adam** [comment]
I am also looking at adding zip drive, DVD, and sdcard.

**Comment by bell07** [comment]
Which capabilities should the new items have ingame?

I think before we can add new capabilities to the laptops, the https://github.com/Gerold55/minetest-laptop/issues/72 needs to be implemented at the first

**Comment by Grizzly-Adam** [comment]
SD Card, like in real life, should function just like usb. CD should function like cd, but not work on somewhat old computers and older. Zip disks were like large floppies, before rewritable cds became the standard. They were not bootable.

**Comment by Grizzly-Adam** [comment]
Tape-- this type of tape isn't for system backup but is used for very old computers to save files. Similar to floppy, but older and slower and absolutely not bootable.

**Comment by bell07** [comment]
As I understand we do not have computers that can use removables "tape" or "punch card"

**Comment by Grizzly-Adam** [comment]
No, not at this time. Some may be old enough, from the looks of it.

**Comment by bell07** [comment]
We need the specs to see which items are usable, which other items missed and which capabilities should be implemented.

**Comment by Grizzly-Adam** [comment]
I am working on specs. See my reply to specs issue. Also been PMing with creator of the PC pack.
